src/Hint/Typecheck.hs

```diff
@@ -37,17 +37,30 @@ kindOf type_expr =
        -- kind of errors
        failOnParseError parseType type_expr
        --
-       kind <- mayFail $ runGhc1 typeKind type_expr
+       (_, kind) <- mayFail $ runGhc1 typeKind type_expr
        --
        kindToString (Compat.Kind kind)
 
+-- | Returns a string representation of the normalized type expression.
+-- This is what @:kind!@ GHCi command prints after @=@.
```

Comment:
After @gelisam's comment, please change to:

`This is what the @:kind!@ GHCi command prints after @=@.`.